README.md

```diff
@@ -1,149 +1,199 @@
+* PORT: Port for running the token refresher on.
+  Defaults to `8085`
+* IMAGE_TAG: Image tag of the [token-refresher image](https://quay.io/repository/rhoas/mk-token-refresher?tab=tags).
+  Defaults to `latest`
+* ISSUER_URL: URL of your auth issuer.
+  Defaults to `https://sso.redhat.com/auth/realms/redhat-external`
+* OBSERVATORIUM_URL: URL of your Observatorium instance.
+  Defaults to `https://observatorium-mst.api.stage.openshift.com/api/metrics/v1/managedkafka`
 
 ## Running the Service locally
+
 Please make sure you have followed all of the prerequisites above first.
 
 1. Compile the binary
-```
-make binary
-```
-2. Clean up and Creating the database
-    - If you have db already created execute
-    ```
-    make db/teardown
-    ```
-    - Create database tables
-    ```
-    make db/setup && make db/migrate
-    ```
-    - Optional - Verify tables and records are created
-    ```
-    make db/login
-    ```
-    ```
-    # List all the tables
-    serviceapitests# \dt
-                       List of relations
-    Schema |        Name        | Type  |       Owner
-    --------+--------------------+-------+-------------------
-    public | clusters           | table | kas_fleet_manager
-    public | connector_clusters | table | kas_fleet_manager
-    public | connectors         | table | kas_fleet_manager
-    public | kafka_requests     | table | kas_fleet_manager
-    public | leader_leases      | table | kas_fleet_manager
-    public | migrations         | table | kas_fleet_manager
-    ```
-
-3. Start the service
-    ```
-    ./kas-fleet-manager serve
-    ```
-    >**NOTE**: The service has numerous feature flags which can be used to enable/disable certain features of the service. Please see the [feature flag](./docs/feature-flags.md) documentation for more information.
-4. Verify the local service is working
-    ```
-    curl -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas
-   {"kind":"KafkaRequestList","page":1,"size":0,"total":0,"items":[]}
-    ```
+     ```
+     make binary
+     ```
+1. Clean up and Creating the database
+     * If you have db already created execute
+       ```
+       make db/teardown
+       ```
+     * Create database tables
+       ```
+       make db/setup && make db/migrate
+       ```
+     * (optional) Verify tables and records are created
+       ```
+       make db/login
+       ```
+       ```
+       # List all the tables
+       serviceapitests# \dt
+                          List of relations
+       Schema |        Name        | Type  |       Owner
+       --------+--------------------+-------+-------------------
+       public | clusters           | table | kas_fleet_manager
+       public | connector_clusters | table | kas_fleet_manager
+       public | connectors         | table | kas_fleet_manager
```

Comment:
```suggestion
```

Reply:
This PR is about formatting changes but I will change it anyway :+1:

Reply:
Thanks.

Reply:
Changed